src/sas/sascalc/dataloader/manipulations.py

```diff
@@ -747,12 +800,14 @@ class _Sector(object):
     Phi is defined between 0 and 2*pi in anti-clockwise
     starting from the x- axis on the left-hand side
     """
-    def __init__(self, r_min, r_max, phi_min=0, phi_max=2 * math.pi, nbins=20):
+
+    def __init__(self, r_min, r_max, phi_min=0, phi_max=2 * math.pi, nbins=20, base = None):
         self.r_min = r_min
         self.r_max = r_max
         self.phi_min = phi_min
         self.phi_max = phi_max
         self.nbins = nbins
+        self.base = base
```

Comment:
This was done only for scripting. Thus no input checking. I'm a great fan of _design by contract_ avoiding `if` in the core methods for performance issues. The validation is indeed better in the calls. I should have documented better though with acceptable values for the base...
It would be great if you added this in GUI with the batch slicer you are doing (the validation for the base goes here!).
I did not write a test because the current tests are based on files with linear binning. I can try to test generate some binning in scipy and cross validate. I will update the PR accordingly.